### DISCUSSION TITLE
Welcome to VirtualFCS Discussions!

### DISCUSSION THREAD

**Discussion by jsimonclark** [Virtual-FCS/VirtualFCS#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jsimonclark** [comment]
Hello everyone, my name is Dr. Simon Clark. I am a research scientist at SINTEF in Norway, and I am one of the developers of the VirtualFCS library. I hope this will be a useful resource for the community to efficiently design and develop fuel cell systems!

**Reply by Yahiaassam** [reply]
Good morning sir,
I allow myself to contact you after having watched your tutorial on the polarization of the Fuel Cell for a study project, I tried to reproduce it at home because there is no complete example, but I cannot correct the problems of the simulation, so I would like to know if you have the complete model.
I also became aware of the versions to use by considering the 1.1 tutorial

**Reply by rakayash** [reply]
Hi Yahiaaassam,
The model does work on OpenModelica v1.14 with Modelica standard library v3.2.3. The fuel cell model cannot be simulated on OpenModelica v1.19. If the version is not the case then please attach your code file with output file.

**Reply by kvid** [reply]
Update: [VirtualFCS v2.0.0](https://github.com/Virtual-FCS/VirtualFCS/releases/tag/v2.0.0), supporting OpenModelica v1.20 for Windows, has now been released.

**Comment by KyrreSundseth** [comment]
Hello, my name is Dr. Kyrre Sundseth. I am a research manager at SINTEF in Norway and I am the coordinator of the FCHJU Virtual-FCS project. Looking forward to interesting VirtualFCS discussions!

**Comment by mikegerhardt** [comment]
Hi, I'm Mike, sometimes known as Dr. Michael R. Gerhardt. I am a research scientist at SINTEF Industry working on developing fuel cell models for the Virtual-FCS library. I am **not** the [professor of jurisprudence](https://law.unc.edu/people/michael-j-gerhardt/) at the University of North Carolina with almost the same name. I'm excited to contribute to this project and develop new tools for everyone to use.

**Comment by NadiaSteiner** [comment]
Hello Everyone!!
My name is Nadia Yousfi Steiner, Professor at the University Bourgogne Franche Comté. I have been working on durability of fuel cell systems for a while. I am Very happy to be part of this project, working on degradation models!

**Comment by rakayash** [comment]
Hello Users, 
Myself Yash. I am a research scientist at SINTEF Industry investigating and developing Hydrogen and Battery systems. In this project I will be codeveloping the models. Looking forward to interesting issues to debug B) Happy Coding!

**Comment by Yahiaassam** [comment]
Hello, my name is ASSAM Yahia, I am an intern at Alstom transport, I am in charge of developing a dynamic behavior model of H2 storage during the operation of the fuel cell.
